charts/gridproxy/Chart.yaml

```diff
@@ -20,4 +20,4 @@ version: 1.1.1
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.7.0-rc1
+appVersion: v1.7.0-rc1
```

Comment:
semver in the build does not take the 'v' into account I think

Reply:
removing the `v` will generate the docker image tag without it, and there was a request to unify all the tags to start with `v` to be more consistent across all projects.

to do that i needed to change the pattern to be `raw` in the release workflow
https://github.com/threefoldtech/tfgridclient_proxy/blob/0ef41af25e056a3aafefc5752834bc124285437a/.github/workflows/release.yml#L31

according to `metadata-action` documentation [here](https://github.com/docker/metadata-action#tags-input:~:text=the%20following%20expressions%3A-,raw%20%3B%20the%20actual%20tag,-version%20%3B%20shorthand%20for) the pattern raw will accept `v` and `rc` also